optaplanner-core/src/test/java/org/optaplanner/core/impl/solver/termination/AndCompositeTerminationTest.java

```diff
@@ -74,27 +74,33 @@ public void calculateSolverTimeGradientTest() {
 
         when(termination1.calculateSolverTimeGradient(solverScope)).thenReturn(0.0);
         when(termination2.calculateSolverTimeGradient(solverScope)).thenReturn(0.0);
+        //min(0.0,0.0) = 0.0
         assertEquals(0.0, compositeTermination.calculateSolverTimeGradient(solverScope), 0.0);
 
         when(termination1.calculateSolverTimeGradient(solverScope)).thenReturn(0.5);
         when(termination2.calculateSolverTimeGradient(solverScope)).thenReturn(0.0);
+        //min(0.5,0.0) = 0.0
         assertEquals(0.0, compositeTermination.calculateSolverTimeGradient(solverScope), 0.0);
 
         when(termination1.calculateSolverTimeGradient(solverScope)).thenReturn(0.0);
         when(termination2.calculateSolverTimeGradient(solverScope)).thenReturn(0.5);
+        //min(0.0,0.5) = 0.0
         assertEquals(0.0, compositeTermination.calculateSolverTimeGradient(solverScope), 0.0);
 
-        when(termination1.calculateSolverTimeGradient(solverScope)).thenReturn(0.1);
-        when(termination2.calculateSolverTimeGradient(solverScope)).thenReturn(0.1);
-        assertEquals(0.1, compositeTermination.calculateSolverTimeGradient(solverScope), 0.0);
+        when(termination1.calculateSolverTimeGradient(solverScope)).thenReturn(-1.0);
+        when(termination2.calculateSolverTimeGradient(solverScope)).thenReturn(-1.0);
+        //negative number are unsupported and ignored, min(unsupported,unsupported) = 1.0 (default)
```

Comment:
Code style: Normally our comments have a space after // and start with a capital. So "// Negative ..."